internal/db/update/schema.go

```diff
@@ -37,8 +37,8 @@ func (s *SchemaUpdate) Check(check schema.Check) {
 	s.check = check
 }
 
-func (s *SchemaUpdate) Version() int {
-	return len(s.updates)
+func (s *SchemaUpdate) Version() (int, int) {
```

Comment:
Doc block or named var names need to explain what the 2 ints returned here are.

Reply:
Or perhaps return a [2]int array so you can treat the combined version as  a single data type?